content/vendor/cli/app ls.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-09-10
+linktitle: "app ls"
+title: "app ls"
+weight: 90105
+draft: false
+---
+
+List all created kots apps, or one in particular.
+
+### Usage
+```bash
+replicated app ls
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
+
+### Examples
+```bash
```

Comment:
To split code blocks add the below line text before the block...

List all created kots apps: